pyprep/find_noisy_channels.py

```diff
@@ -480,19 +531,22 @@ def find_bad_by_ransac(
 
         References
         ----------
-        .. [1] Fischler, M.A., Bolles, R.C. (1981). Random rample consensus: A
+        .. [1] Fischler, M.A., Bolles, R.C. (1981). Random sample consensus: A
```

Comment:
ahhh yes, the RANRAC :)

Reply:
lol ever since I caught that one I can't help but read it in the Scooby-Doo voice